### ISSUE TITLE
Support for optional fields in table schema

### ISSUE THREAD

**Issue by Squix** [domdomegg/airtable-ts#1]
Hi, thank you for your amazing work !
It really helps to have this wrapper for the clunky airtable api.

Unfortunately, I can't use it fully, because I wasn't able to understand how to pass fields as optionals in the schema.
I need optionals fields for the insert operation.

Here's my table :

```(js)
const tableA = {
      name:"Demandes",
      baseId
      tableId:
      schema: {req_id:'string | undefined', foo:string},
      mappings: {req_id:'fldvND4TXCpoIvXxl', foo:"flxblabla"}
    } as Table<{id:string, req_id:string | undefined, foo:string}>,
```

I want _req_id_ to be optional. I tried using "string | undefined" but I get the error _Type '"string | undefined"' is not comparable to type '"string"'_.

And my insert operation : 

```(js)
await db.insert(tableA, {
         foo:"truc"
        })
```

Here he's complaining at runtime that req_id is not included : _[airtable-ts] Item for Demandes table is missing field 'req_id' (expected string)"_

I tried setting req_id to an empty string, then I get _[airtable-ts] formula type field is readonly_ (req_id is a formula, so it should not be in an insert operation).

Thank you for your help !

**Comment by domdomegg** [comment]
Hi, I think regarding optional fields you'll want to define them as `string | null`, for example

```ts
const tableA = {
      name: "Demandes",
      baseId: "app123",
      tableId: "tbl123",
      schema: {req_id:'string | null', foo:string},
      mappings: {req_id:'fldvND4TXCpoIvXxl', foo:"flxblabla"}
} as Table<{id:string, req_id:string | null, foo:string}>,
```

On inserts not allowing you to omit properties, I've fixed this in the most recent version (1.3.0) - apologies for this! You can now use:

```ts
await db.insert(tableA, {
  foo:"truc"
})
```

**Comment by Squix** [comment]
Thanks ! It works.
